### PR TITLE
[Impro](regression-test) More test cases for function round

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**
Add test case to for function `round` when using float & decimal as input value type

### CODE REVIEW

**Review by HappenLee — DISMISSED** [review]
LGTM

File: regression-test/suites/query_p0/sql_functions/math_functions/test_round.groovy [hunk 1]
```diff
@@ -21,6 +21,11 @@
     qt_select "SELECT round_bankers(10.12345)"
     qt_select "SELECT round_bankers(10.12345, 2)"
 
+    // test tie case 1: float, banker's rounding
+    qt_select "SELECT number*10/100, ROUND(number * 10 / 100) from NUMBERS('number'=50) where number % 5 = 0;"
```
Comment (yiguolei) [inline regression-test/suites/query_p0/sql_functions/math_functions/test_round.groovy:25]: add order by please. Or the result is not stable, and the workflow may failed.

File: regression-test/suites/query_p0/sql_functions/math_functions/test_round.groovy [hunk 2]
```diff
@@ -21,6 +21,11 @@
...
+    // test tie case 2: decimal, rounded up when tie
+    qt_select "SELECT number*10/100, ROUND(CAST(number * 10 AS DECIMALV3(10,2)) / 100) from NUMBERS('number'=50) where number % 5 = 0"
```
Comment (yiguolei) [inline regression-test/suites/query_p0/sql_functions/math_functions/test_round.groovy:27]: and here

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**